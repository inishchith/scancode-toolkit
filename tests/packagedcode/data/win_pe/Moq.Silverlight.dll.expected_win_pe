{
  "APIVersion": "", 
  "Assembly Version": "4.2.1507.118", 
  "BuildDate": "", 
  "BuildVersion": "", 
  "Comments": "", 
  "Company": "", 
  "CompanyName": "Clarius Consulting, Manas Technology Solutions, InSTEDD", 
  "FileDescription": "Moq", 
  "FileVersion": "4.2.1507.0118", 
  "Full Version": "", 
  "InternalName": "Moq.Silverlight.dll", 
  "LegalCopyright": "", 
  "LegalTrademarks": "", 
  "LegalTrademarks1": "", 
  "LegalTrademarks2": "", 
  "License": "", 
  "OriginalFilename": "Moq.Silverlight.dll", 
  "ProductName": "Moq", 
  "ProductVersion": "4.2.1507.0118", 
  "WWW": "", 
  "extra_data": {}
}