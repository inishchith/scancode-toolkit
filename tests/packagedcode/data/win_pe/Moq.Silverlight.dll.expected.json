{
  "APIVersion": null, 
  "Assembly Version": "4.2.1507.118", 
  "BuildDate": null, 
  "BuildVersion": null, 
  "Comments": null, 
  "Company": null, 
  "CompanyName": "Clarius Consulting, Manas Technology Solutions, InSTEDD", 
  "FileDescription": "Moq", 
  "FileVersion": "4.2.1507.0118", 
  "Full Version": null, 
  "InternalName": "Moq.Silverlight.dll", 
  "LegalCopyright": "", 
  "LegalTrademarks": null, 
  "LegalTrademarks1": null, 
  "LegalTrademarks2": null, 
  "License": null, 
  "OriginalFilename": "Moq.Silverlight.dll", 
  "ProductName": "Moq", 
  "ProductVersion": "4.2.1507.0118", 
  "WWW": null, 
  "extra_data": {}
}